index.d.ts

```diff
@@ -522,7 +527,7 @@ declare namespace WAWebJS {
         /** Message type */
         type: MessageTypes,
         /** Links included in the message. */
-        links: string[],
+        links: Array<linkStructure>,
```

Comment:
Since it's not being used anywhere else, we can probably do this inline
```suggestion
        links: Array<{
            link: string,
            isSuspicious: boolean
        }>,
```